### ISSUE TITLE
`jaeger-cassandra-schema` image not working

### ISSUE THREAD

**Issue by yurishkuro** [jaegertracing/jaeger#244]
@jpkrohling I was trying to test the changes in #243 that, in part, affect the `jaeger-cassandra-schema` image, and I could not get it to work with Cassandra 3.9 image. It seems the base image `jpkroehling/cassandra` that provides CQLSH is not compatible with 3.9:

```
$ docker run -e CQLSH_HOST=192.168.1.4 -e CASSANDRA_WAIT_TIMEOUT=10 --network host 5bb838365a50
Connection error: ('Unable to connect to any servers', {'192.168.1.4': ProtocolError("cql_version '3.4.0' is not supported by remote (w/ native protocol). Supported versions: [u'3.4.2']",)})
Cassandra is still not up at 192.168.1.4. Waiting 1 second.
```

It looks like the base image is for v3.0.12, which in fact is not currently compatible with Jaeger schema (#189).

**Comment by yurishkuro** [comment]
Also, I could not find any place in the documentation that mentions using this image. The [Deployment](http://jaeger.readthedocs.io/en/latest/deployment/#cassandra) page suggests using the source script `sh ./plugin/storage/cassandra/cassandra3v001-schema.sh test | cqlsh`, it would be good to (at least) add the instructions on how to use the Docker image.

**Comment by objectiser** [comment]
Juca is away until 11th on PTO without any/much internet access.

For now it may be better to use the official cassandra image (https://hub.docker.com/_/cassandra/) - the main reason that Juca was using a local image was to include a bug fix not currently available in the official image. However I believe this bug only affects large scale deployments.

**Comment by jpkrohling** [comment]
This should have been fixed by #278 .

**Comment by pavolloffay** [comment]
Fixed closing.
